ui/src/components/Publication/Creation/KeyInformation.tsx

```diff
@@ -13,6 +13,23 @@ import * as Types from '@types';
 const KeyInformation: React.FC = (): React.ReactElement => {
     const title = Stores.usePublicationCreationStore((state) => state.title);
     const updateTitle = Stores.usePublicationCreationStore((state) => state.updateTitle);
+
+    const affiliationsStatus = Stores.usePublicationCreationStore(
```

Comment:
All these lines could be simplified by using destructuring assignment. Please update the old assignments like this as well (within this file). This looks like a lot of code for so little, just to grab a bunch of properties from the store.